github/copilot.go

```diff
@@ -63,6 +64,36 @@ type SeatCancellations struct {
 	SeatsCancelled int `json:"seats_cancelled"`
 }
 
+type CopilotUsageSummaryListOptions struct {
```

Comment:
All new exported structs need Godoc-style comments complete with punctuation (here and on lines 74 and 84 below).

For example:

```go
// CopilotUsageSummaryListOptions specifies optional parameters to the
// CopilotService.GetOrganizationUsage method.
```